posts/views.py

```diff
@@ -14,11 +16,14 @@ def post(self, request):
         try:
             title = request.data['title']
             content = request.data['content']
-            author = request.author
-            post = Post.objects.create(title=title, content=content, user_id=author)
+            user_id = request.data['user_id']
```

Comment:
근데 이렇게 하면 보내는쪽도 user_id로 보내야할 것 같은데, 명세만 맞추면 될 것 같아요. 어떻게 보내고 어떻게 받을건지
```suggestion
            user_id = request.data['user_id']
```